### PR TITLE
Add 2wt tooltip infos

### PR DESCRIPTION
**Pull request by archendel**

### CODE REVIEW

**Review by AbdelHedhili — COMMENTED** [review]
Probably better not to do it in this PR but I think it's time we add a tooltip type to get exactly the info we need in the tooltips without having them in common with the MAP type (which might not need the same infos). We couldn't do it before because we didn't have the LOD that we now have.

File: src/main/java/org/gridsuite/network/map/dto/twowindingstransformer/TwoWindingsTransformerMapInfos.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.network.extensions.BranchStatus;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+import org.gridsuite.network.map.model.CurrentLimitsData;
+
+@SuperBuilder
+@Getter
+public class TwoWindingsTransformerMapInfos extends AbstractTwoWindingsTransformerInfos {
```
Comment (etiennehomer) [inline src/main/java/org/gridsuite/network/map/dto/twowindingstransformer/TwoWindingsTransformerMapInfos.java:12]: You also need the 2 voltage levels the 2WT is attached to

File: src/main/java/org/gridsuite/network/map/dto/line/LineTooltipInfos.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+    private String voltageLevelId1;
+
+    private String voltageLevelId2;
+
+    private Boolean terminal1Connected;
+
+    private Boolean terminal2Connected;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```
Comment (etiennehomer) [inline src/main/java/org/gridsuite/network/map/dto/line/LineTooltipInfos.java:36]: You don't need p1 and p2 nope ?

**Review by etiennehomer — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/network/map/dto/twowindingstransformer/TwoWindingsTransformerTooltipInfos.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
+package org.gridsuite.network.map.dto.twowindingstransformer;
```
[the remark this answers is not in the document]
Comment (archendel) [reply src/main/java/org/gridsuite/network/map/dto/twowindingstransformer/TwoWindingsTransformerTooltipInfos.java:8]: fixed

File: src/main/java/org/gridsuite/network/map/dto/line/LineTooltipInfos.java (hunk 2 above)
Comment (archendel) [reply src/main/java/org/gridsuite/network/map/dto/line/LineTooltipInfos.java:36]: fixed

Comment (archendel) [reply src/main/java/org/gridsuite/network/map/dto/line/LineTooltipInfos.java:36]: removed in TwoWindingsTransformerTooltipInfos.java as well

**Review by AbdelHedhili — APPROVED** [review]

**Review by etiennehomer — APPROVED** [review]

**Pull request merged**